stock_available_to_promise_release/models/stock_route.py

```diff
@@ -7,6 +7,16 @@
 class StockRoute(models.Model):
     _inherit = "stock.route"
 
+    allow_unrelease_return_done_move = fields.Boolean(
```

Comment:
IMHO this introduce too big functions for return management here. A new module with only the good entry points here could be better